res/http-export/configuration.toml

```diff
@@ -97,18 +97,18 @@ Port = 6379
 Timeout = "30s"
```

Comment:
need to remove `ConfigAccessTokenFile ` and `AccessTokenFile ` from the above config.